docs/guides/development/nodejs/nodejs-twitter-bot/index.md

```diff
@@ -0,0 +1,294 @@
+
+	        // Post that tweet
+	        T.post('statuses/update', { status: replyText, in_reply_to_status_id: id }, gameOver);
+
+	    } else {
+	        console.log("uh-uh-uh, they didn't say the magic word.");
+	    };
+
+	    function gameOver(err, reply) {
+	        if (err) {
+	            console.log(err.message);
+	            console.log("Game Over");
+	        } else {
+	            console.log('Tweeted: ' + reply.text);
+	        }
+	    };
+	}
+```
+</br>
+
+`pressStart` contains a few local variables, a bit of logic, and a final function that must be included in the `T.post` method. You can use an unnamed function there and it will do the same thing, but I went ahead and wrote on separately for readability. Essentially, the function `gameOver` gives us a chance to log an error if it occurs or to log the tweet that was sent out.
+
+`pressStart` takes “tweet” as a parameter. This is the tweet that another user writes that tags SnesSoundtracks. That tweet has tons of data attached to it, data that Twit helps us parse through. The first three variables are the id of the tweet, the text of the tweet, and the username of the person who wrote the tweet. We will need those three in order to respond accurately as a comment to the original tweet by the other user.
+
+Up next is a regex for whatever word you want to activate the bot to reply. I chose “please." As long as the user is polite, they’ll get a random soundtrack.
+
+The regex has “g” and “i” at the end so it ignores capitalization and checks globally for the word please. `playerOne` is a variable that can either be an empty array or will use `.match` to create an array with one element: the word “please”. `playerTwo` is a boolean that verifies whether the array `playerOne` has anything in it.
+
+The if statement requires that the tweet text contains the bot’s name and an array of at least one element was passed into `playerTwo`. If both of these come back as true, then we proceed to an area where the variable `replyText` is created, which includes a random element of the array, as well as the username of the person being replied to and a short message. `replyText` is passed into an object that contains two properties: `status` and `in_reply_to_status_id`. `status` is the actual text to be posted to the tweet, in our case the variable `replyText` is our status. `In_reply_to_status_id` is defined as `id`, which is a variable from the beginning of the `pressStart` function. `id` is a unique identifier of a tweet from Twitter’s API. This allows Twit to identify which tweet the bot will reply to as a comment. Finally, the else statement at the end will log a quote from Jurassic Park to the console if the user doesn’t say please, just so we can see why the bot didn't tweet back.
+
+```javascript
+	function pressSelect() {
+
+	    var weeklyReplyText = soundtrackArray[soundtrackArrayElement] + " Here's your soundtrack for the week!";
```

Comment:
I renamed the variable `weeklyReplyText` to `weeklyText`, because it wasn't in reply to another Tweet, so I thought that would remove some potential ambiguity for the reader

I also appended the date and time the Tweet is sent to the scheduled tweet's text:
https://deploy-preview-4492--nostalgic-ptolemy-b01ab8.netlify.app/docs/guides/nodejs-twitter-bot/#scheduling-tweets

I did this because I was getting `Status is a duplicate` errors from the API client when testing the code, and I didn't want readers to run into that. 

Twitter's API seems to prevent you from sending identical tweets in succession or within some period of time (that I couldn't find an exact definition for): https://stackoverflow.com/questions/36971860/duplicate-status-error-when-tweeting-using-twitter-api

By including the date and time, the tweet text varies by a little bit, and the API error disappeared.

Reply:
That's a good way to keep the duplicate status problem from happening.